romancal/ramp_fitting/ramp_fit_step.py

```diff
@@ -216,9 +221,17 @@ def ols_cas22(self, input_model, readnoise_model, gain_model):
         var_rnoise = output.variances[..., Variance.read_var]
         var_poisson = output.variances[..., Variance.poisson_var]
         err = np.sqrt(var_poisson + var_rnoise)
+        dq = output.dq.astype(np.uint32)
+
+        ramp_dq = np.zeros(dq.shape[1:], dtype=np.uint32)
+        # Turn the jump detection DQ into a ramp DQ
+        for i in range(dq.shape[1]):
+            for j in range(dq.shape[2]):
+                if np.any(dq[:, i, j] == dqflags.group["JUMP_DET"]):
+                    ramp_dq[i, j] = dqflags.group["JUMP_DET"]
```

Comment:
I was imagining ramp_dq |= np.any(dq, axis=0) * dqflags.group['JUMP_DET'].
I hadn't appreciated that the new ramp fitting didn't include any other DQ flags.  We should add at least SATURATION and DONOTUSE or something back, but I'll review what needs to happen there and we shouldn't make it part of this PR.